### DISCUSSION TITLE
Forge port and optifine features

### DISCUSSION THREAD

**Discussion by BSFGP** [xCollateral/VulkanMod#254] [Q&A] [answered]
1. Can we expect a port of this mod for the Forge modloader? Fabric is not an ultimate modloader and Forge is still quite popular
2. Can we expect Optifine features (such as shader support, dynamic lighting, custom/random textures and models for blocks and entities etc.)? This could make this mod a one-stop solution, completely replacing optifine and sodium

**Comment by sqrvrt** [comment, marked as answer]
1. You're asking to rewrite X in Rust, but mc equivalent. It's tedious, it doesn't pay off. If it ever will be made it most certainly wouldn't be the same dev.
2. It seems you don't understand shaders very well; the ones written for optifine are practically incompatible with vulkan, and would require rewriting them from scratch. Other features could definitely be done, but again it isn't focus of this mod; if you're intrested in doing it yourself i would encourage you to start it.
